crossing-training.lic

```diff
@@ -53,6 +56,7 @@ class CrossingTraining
     @equipment_manager = EquipmentManager.new(@settings)
 
     UserVars.crossing_timers ||= {}
+    UserVars.chadatru_prayer_last ||= Time.now - 3600
```

Comment:
Should this be `... - @chadatru_prayer_cooldown`?

Reply:
No, so the chadatru is a one hour cooldown, and we can keep track of it so if we start up we do it automatically, but if we restart we don't waste time doing it again. The cooldown checks that it's been an hour since the `chadatru_prayer_last` time. The only time there can be an overlap is on initial run, where if you've run it recently, update the scripts and get this functionality, and then run it you will overlap, but it shouldn't be an issue. Or if you delete that UserVar you will run into the previous issue.

Reply:
I'm not sure I was clear: what I was suggesting is `UserVars.chadatru_prayer_last ||= Time.now - @chadatru_prayer_cooldown`.

Why use the magic number '3600' here when `@chadatru_prayer_cooldown` is defined above to be '3600'. Are these two logically unrelated and just happen to be the same value?

Reply:
Happen to be the same value, the 3600 was added before the variable. Yup, I'll change it.

Reply:
Except sometimes magic numbers make me feel like a magician? 🥇